cli/utils/response.go

```diff
@@ -1,4 +1,4 @@
-package cli
+package utils
```

Comment:
As of this writing there are three files in utils/:
- dcoscli.go: access config data in the main CLI, run main CLI commands
- http.go: build requests to query the scheduler (and soon likely other services in the cluster)
- response.go: handle http responses

IMO two changes could be made here:
- response.go could be folded into http.go as they're pretty closely related (I think http.go even invokes response.go)
- I'm a bit hesitant to have a `utils` package in general. I think @benclarkwood 's point of instead calling it `client` or `clients` would work, as the functionality here is basically client code for querying 1) schedulers over HTTP and 2) the main DC/OS CLI over the commandline.